api/src/main/java/org/eclipse/microprofile/opentracing/Traced.java

```diff
@@ -31,21 +31,51 @@
 import static java.lang.annotation.RetentionPolicy.RUNTIME;
 
 /**
- * Define the InterceptorBinding that wraps methods to be traced.
+ * This annotation applies to a class or a method. When applied to a class, this
```

Comment:
Maybe add a short description at the beginning and then this paragraph

Reply:
Done - added intro summary